tests/base.py

```diff
@@ -17,17 +17,16 @@ class FreshdeskBaseTest(unittest.TestCase):
     INCREMENTAL = "INCREMENTAL"
     FULL = "FULL_TABLE"
 
-    START_DATE_FORMAT = "%Y-%m-%dT00:00:00Z" # %H:%M:%SZ
-    BOOKMARK_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+    start_date = ""
```

Comment:
Is `start_date` vaiable with blank string really required?

Reply:
Yes, in the start_date test case we override the start date using this variable.